integration/testcmd/generic/uinit/generic.go

```diff
@@ -0,0 +1,69 @@
+// Copyright 2019 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package main
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"log"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+
+	"github.com/u-root/u-root/pkg/mount"
+	"golang.org/x/sys/unix"
+)
+
+func getCommand(testCmd string) *exec.Cmd {
+	split := strings.Split(testCmd, " ")
```

Comment:
I'd try to get rid of this particular part.

Imagine the following TestCmd: `kexec --module="./foo bar" --module=zap` which should end up being `[]string{"kexec", "--module=\"./foo bar\"", "--module=zap"}`

By doing this, you've signed up to do shell tokenization. We kind of unsuccessfully did the same in the original code here: https://github.com/u-root/u-root/blob/671f180a127774f6080018fd55be401bc98d4949/pkg/vmtest/fields.go#L22

This is why having elvish was actually kind of nice -- it's built in.

Reply:
I like json as a goal, but elvish actually does this job. Let's file a bug for json later, and keep elvish for now?

Reply:
yep

Reply:
Done!